src/ansys/geometry/core/connection/conversions.py

```diff
@@ -431,3 +447,152 @@ def grpc_frame_to_frame(frame: GRPCFrame) -> Frame:
             ]
         ),
     )
+
+
+def grpc_surface_to_surface(surface: GRPCSurface, surface_type: "SurfaceType") -> Surface:
+    """
+    Convert an ``ansys.api.geometry.Surface`` gRPC message to a ``Surface`` class.
```

Comment:
Here, you follow ``Surface`` with class, but you don't do this in later descriptions. Should they all end in class or perhaps based on the return description of later ones, perhaps follow the code entity with some other noun (such as message). It's helpful (and follows our guidelines) to have an object type after the code entity.